templates/deployment.yaml

```diff
@@ -9,6 +9,10 @@ metadata:
   name: {{ include "krakend.fullname" . }}
   labels:
     {{- include "krakend.labels" . | nindent 4 }}
+  {{- if .Values.deploymentAnnotations }}
```

Comment:
you might want to follow the same pattern we've been using:

```
  {{- with .Values.deploymentAnnotations }}
    annotations:
    {{- toYaml . | nindent 4 }}
  {{- end }}
```

for consistency.